app/src/main/java/org/mozilla/fenix/settings/address/view/AddressEditorView.kt

```diff
@@ -36,26 +43,59 @@ class AddressEditorView(
         binding.saveButton.setOnClickListener {
             saveAddress()
         }
+
+        address?.let { address ->
+            binding.saveButton.text =
+                binding.root.context.getString(R.string.addressess_update_address_button)
```

Comment:
Let's not make the save button text change yet. We don't currently do this for "Edit card" so I have noted the inconsistency with UX.

Reply:
Made the change. IMO we should have also changed the button for CC, having it say: "update" or "save changes". It would be more consistent with the editing action. But let's see what UX decides.

Reply:
I did mention how it was inconsistent to Kate yesterday. It would be very likely that we want to make the changes everywhere, but the strings aren't final yet. So, that's where we are with this.